examples/multi-cursors/.env.local.example

```
@@ -1,3 +1,3 @@
-VITE_CURSOR_FEED_ID=example-cursor-feed-id
-VITE_GROUP_ID=co_example-group-id
-VITE_OLD_CURSOR_AGE_SECONDS=5
\ No newline at end of file
+VITE_CURSOR_FEED_ID=multi-cursors-250425-1708
+VITE_GROUP_ID=co_zXE8C8sd9QxEbxnt3neRvFRPFUc
+VITE_OLD_CURSOR_AGE_SECONDS=36000
```

```suggestion
VITE_OLD_CURSOR_AGE_SECONDS=5
```

I think it was correct having 5 seconds as default